tldextract/tldextract.py

```diff
@@ -374,12 +374,13 @@ def main():
     parser.add_argument('-c', '--cache_file', help='use an alternate TLD definition file')
 
     args = parser.parse_args()
+    tld_extract = TLDExtract(include_psl_private_domains=True)
```

Comment:
> For the 1.x series though, I think I'd be happy with include_psl_private_domains being configurable here.

Bump. Sorry if it wasn't clear, this is the only blocker on this PR. Should be an easy update.

(I don't think anybody's gonna tackle #66 anytime soon. Don't hold your breath for it.)